plugins/modules/remote_management/redfish/ilo_redfish_config.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021-2022 Hewlett Packard Enterprise, Inc. All rights reserved.
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ilo_redfish_config
+short_description: Sets or updates a configuration attribute. For use with HPE iLO operations that require Redfish OEM extensions.
+description:
```

Comment:
But using `version_added` for the module itself is mandatory:
```suggestion
version_added: 4.1.0
description:
```